static/easydoc.js

```diff
@@ -1,36 +1,77 @@
+// Plugged into <a> elements from the table of contents menu
 function tocClick(topUrl, contentUrl) {
     var $content = $('#content');
     $content.attr('src', contentUrl)
     window.location.replace(topUrl)
     return false
 }
 
+function setCssOpenClass($el, isOpen) {
+    if (isOpen) {
+        $el.removeClass('closed');
+        $el.addClass('open');
+    } else {
+        $el.removeClass('open');
+        $el.addClass('closed');
+    }
+}
+
+function toggleCssOpenClass($el) {
+    if ($el.hasClass('open')) {
+        setCssOpenClass($el, false);
+    } else {
+        setCssOpenClass($el, true);
+    }
+}
+
 $(function() {
-    // href clicking in the TOC
     var $content = $('#content');
-    $('#toc a')
-        .click(function(topUrl, contentUrl) {
-            return false;
-        });
+    var $toc = $('#toc');
+    var $tocDrawerToggle = $('#toc-drawer-toggle')
+    var $searchBox = $('#search-field');
+    var $searchButton = $('#search-field-button');
 
+    addMediaSpecificBehavior();
+
+    // href clicking in the TOC
+    $toc.find('a').click(function(topUrl, contentUrl) {
```

Comment:
Can probably just handle the toc clicks here instead of using an html attribute onclick for the <a> elements.